### PR TITLE
[docs] Describe bypassing the proxy

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @AlexKamaev

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -472,6 +473,26 @@ You can also specify authentication credentials with the proxy host.
...
 
+### --proxy-bypass \<rules\>
+
+Specifies the resources accessed bypassing the proxy server.
+
+When you access the Internet through a proxy server specified using the [--proxy](#--proxy-host) option, you may still need some local or external resources to be accessed directly. In this instance, provide their URLs to the `--proxy-bypass` option.
+
+The `rules` parameter takes a comma-separated list (without spaces) of URLs that require direct access. You can omit parts of the URLs or replace them with wildcards `*` (which has the same effect). TestCafe will correspond these symbols to any number of characters in the URL.
+
+```sh
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: I'd describe shortly what these examples mean.

File: docs/articles/documentation/using-testcafe/programming-interface/runner.md [hunk 2]
```diff
@@ -361,7 +368,11 @@ runner.useProxy('proxy.corp.mycompany.com');
 ```
 
 ```js
-runner.useProxy('172.0.10.10:8080');
+runner.useProxy('172.0.10.10:8080', ['localhost:8080', 'internal-resource.corp.mycompany.com']);
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/programming-interface/runner.md]: Same here

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 3]
```diff
@@ -472,6 +473,32 @@ You can also specify authentication credentials with the proxy host.
...
+testcafe chrome my-tests/**/*.js --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080
+```
+
+In the example below, two resources are accessed bypassing the proxy: `localhost:8080` and `internal-resource.corp.mycompany.com`.
+
+```sh
+testcafe chrome my-tests/**/*.js --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080,internal-resource.corp.mycompany.com
+```
+
+The `*.mycompany.com` value means that all URLs in `mycompany.com` subdomains will be accessed directly.
```
Comment (AlexKamaev) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Should we mention that the rule `*.mycompany.com` is the same as `.mycompany.com` (without *) ? I mean that it is not required to add an asterisk in the beginning of in the end of rule, but it's required in the middle of rule (.mycompany.*.com)

Comment (VasilyStrelyaev) [reply docs/articles/documentation/using-testcafe/command-line-interface.md]: Yeah, I didn't thing about wildcards in the middle

**Comment by kirovboris** [comment]
ping @DevExpress/testcafe-docs

**Comment by VasilyStrelyaev** [comment]
@dirk-pieterse , please check the docs

**Pull request merged**